purescript.cabal

```diff
@@ -119,66 +119,66 @@ common defaults
     -- specific version.
     aeson >=1.5.6.0 && <1.6,
     aeson-better-errors >=0.9.1.0 && <0.10,
-    aeson-pretty >=0.8.8 && <0.9,
-    ansi-terminal >=0.10.3 && <0.11,
+    aeson-pretty >=0.8.9 && <0.9,
+    ansi-terminal ==0.11.*,
     array >=0.5.4.0 && <0.6,
-    base >=4.14.1.0 && <4.15,
+    base >=4.14.3.0 && <4.15,
     base-compat >=0.11.2 && <0.12,
     blaze-html >=0.9.1.2 && <0.10,
     bower-json >=1.0.0.1 && <1.1,
     boxes >=0.1.5 && <0.2,
     bytestring >=0.10.12.0 && <0.11,
     Cabal >=3.2.1.0 && <3.3,
-    cborg >=0.2.4.0 && <0.3,
-    serialise >=0.2.3.0 && <0.3,
+    cborg >=0.2.6.0 && <0.3,
+    serialise >=0.2.4.0 && <0.3,
     cheapskate >=0.1.1.2 && <0.2,
-    clock ==0.8.*,
-    containers >=0.6.2.1 && <0.7,
-    cryptonite ==0.27.*,
+    clock >=0.8.2 && <0.9,
+    containers >=0.6.5.1 && <0.7,
+    cryptonite ==0.29.*,
     data-ordlist >=0.4.7.0 && <0.5,
     deepseq >=1.4.4.0 && <1.5,
     directory >=1.3.6.0 && <1.4,
-    dlist >=0.8.0.8 && <0.9,
+    dlist ==1.0.*,
     edit-distance >=0.2.2.1 && <0.3,
-    file-embed >=0.0.13.0 && <0.1,
+    file-embed >=0.0.15.0 && <0.1,
     filepath >=1.4.2.1 && <1.5,
     fsnotify >=0.3.0.1 && <0.4,
     Glob >=0.10.1 && <0.11,
-    haskeline >=0.8.1.1 && <0.9,
-    language-javascript ==0.7.0.0,
-    lifted-async >=0.10.1.3 && <0.11,
+    haskeline >=0.8.2 && <0.9,
+    language-javascript ==0.7.1.0,
+    lifted-async >=0.10.2.2 && <0.11,
     lifted-base >=0.2.3.12 && <0.3,
     memory >=0.15.0 && <0.16,
-    microlens >=0.4.11.2 && <0.5,
-    microlens-platform >=0.4.1 && <0.5,
-    monad-control >=1.0.2.3 && <1.1,
+    microlens >=0.4.12.0 && <0.5,
+    microlens-platform >=0.4.2 && <0.5,
+    monad-control >=1.0.3.1 && <1.1,
     monad-logger >=0.3.36 && <0.4,
     mtl >=2.2.2 && <2.3,
     parallel >=3.2.2.0 && <3.3,
     parsec >=3.1.14.0 && <3.2,
     pattern-arrows >=0.0.2 && <0.1,
-    process >=1.6.9.0 && <1.7,
+    process >=1.6.13.2 && <1.7,
     protolude >=0.3.0 && <0.4,
     purescript-cst ==0.4.0.0,
-    regex-tdfa >=1.3.1.0 && <1.4,
+    regex-tdfa >=1.3.1.1 && <1.4,
     safe >=0.3.19 && <0.4,
-    scientific >=0.3.6.2 && <0.4,
-    semigroups >=0.19.1 && <0.20,
+    scientific >=0.3.7.0 && <0.4,
+    semigroups >=0.19.2 && <0.20,
     semialign >=1.1.0.1 && <1.2,
-    sourcemap >=0.1.6 && <0.2,
+    sourcemap >=0.1.6.1 && <0.2,
     split >=0.2.3.4 && <0.3,
-    stm >=2.5.0.0 && <2.6,
+    stm >=2.5.0.1 && <2.6,
     stringsearch >=0.3.6.6 && <0.4,
     syb >=0.7.2.1 && <0.8,
     text >=1.2.4.1 && <1.3,
     these >=1.1.1.1 && <1.2,
     time >=1.9.3 && <1.10,
     transformers >=0.5.6.2 && <0.6,
-    transformers-base >=0.4.5.2 && <0.5,
+    transformers-base >=0.4.6 && <0.5,
     transformers-compat >=0.6.6 && <0.7,
-    unordered-containers >=0.2.13.0 && <0.3,
+    unordered-containers >=0.2.14.0 && <0.3,
     utf8-string >=1.0.2 && <1.1,
-    vector >=0.12.1.2 && <0.13
+    vector >=0.12.3.1 && <0.13
```

Comment:
[Nothing of note](https://hackage.haskell.org/package/vector-0.12.3.1/changelog)